tap_dixa/streams/conversations.py

```diff
@@ -19,7 +19,7 @@ class Conversations(IncrementalStream):
     endpoint = "/v1/conversation_export"
 
     # pylint: disable=signature-differs
-    def get_records(self, start_date):
+    def get_records(self, start_date, config: dict = {}):
```

Comment:
is the config param being used in the method? If not, Is it ok to remove

Reply:
yes @kethan1122  this config is required for the page_size attribute which is used for the ActivityLog extraction